docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,194 @@
+# Release 479 (TBD 2025)
+
+## General
+
+* Support automatic TLS certificate generation in the `ANNOUNCE` node discovery mode. ({issue}`27030`)
+* Add support for specifying the location of the Exchange Manager configuration file. ({issue}`26611`)
+* Publish query finishing time in event listener. ({issue}`27202`)
+* Add lineage support for output columns in `SELECT` queries. ({issue}`26241`)
```

Comment:
This is unclear. What's the user-visible impact of this? What does it mean to have lineage for output columns?

Reply:
so, it is more likely :  "Add lineage support for output columns of `SELECT` queries"